specification/purview/data-plane/Azure.Analytics.Purview.DataPolicies/preview/2021-01-01-preview/purviewDataPolicy.json

```diff
@@ -0,0 +1,835 @@
+        "type": "array",
+        "items": {
+          "$ref": "#/definitions/AttributeMatcher"
+        }
+      }
+    },
+    "DecisionRule": {
+      "description": "The decision rule for a policy",
+      "type": "object",
+      "properties": {
+        "kind": {
+          "description": "The kind of rule",
+          "enum": [
+            "decisionrule"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "Kind",
+            "modelAsString": true
+          }
+        },
+        "effect": {
+          "description": "The effect for rule",
+          "enum": [
+            "Deny",
+            "Permit"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "Decision",
+            "modelAsString": true
+          }
+        },
+        "cnfCondition": {
+          "$ref": "#/definitions/CnfCondition"
+        }
+      }
+    },
+    "SystemData": {
+      "type": "object",
+      "properties": {
+        "createdBy": {
+          "type": "string"
+        },
+        "createdByType": {
+          "type": "string"
+        },
+        "createdAt": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "lastModifiedBy": {
+          "type": "string"
+        },
+        "lastModifiedByType": {
+          "type": "string"
+        },
+        "lastModifiedAt": {
+          "format": "date-time",
+          "type": "string"
+        }
+      }
+    },
+    "CollectionReference": {
+      "description": "The collection reference for a policy",
+      "type": "object",
+      "properties": {
+        "type": {
+          "description": "The type of reference",
+          "default": "CollectionReference",
+          "type": "string"
+        },
+        "referenceName": {
+          "description": "The name of reference",
+          "type": "string"
+        }
+      }
+    },
+    "DataFabricPolicyContentDetails": {
+      "type": "object",
+      "properties": {
+        "policyType": {
+          "type": "string"
+        }
+      }
+    },
+    "DataFabricPolicyContent": {
+      "type": "object",
+      "properties": {
+        "description": {
+          "type": "string"
+        },
+        "version": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "decisionRules": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DecisionRule"
+          }
+        },
+        "systemData": {
+          "$ref": "#/definitions/SystemData"
+        },
+        "collection": {
+          "$ref": "#/definitions/CollectionReference"
+        },
+        "policyDetails": {
+          "$ref": "#/definitions/DataFabricPolicyContentDetails"
+        }
+      }
+    },
+    "DataPolicy": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/DataFabricPolicyContent"
+        },
+        "name": {
```

Comment:
is this the displayName then, when there is an id property too?

Reply:
Yes this is the display name. It is an input from the user.